packages/twenty-front/src/pages/settings/data-model/SettingsObjectDetail.tsx

```diff
@@ -92,7 +195,25 @@ export const SettingsObjectDetail = () => {
     });
 
   const shouldDisplayAddFieldButton = !activeObjectMetadataItem.isRemote;
-
+  const tableHeadings: TableHeading = [
+    { label: 'Name', sortKey: SortKeys.label },
+    {
+      label: activeObjectMetadataItem.isCustom ? 'Identifier' : 'Field type',
+      sortKey: SortKeys.fieldType,
+    },
+    {
+      label: 'Data type',
+      sortKey: SortKeys.dataType,
+    },
+  ];
+  const SortIcon = ({ sortKey }: { sortKey: SortKeys }) => {
+    if (sortKey !== sortConfig.fieldName) return null;
+    return sortConfig.orderBy === 'AscNullsLast' ? (
+      <IconArrowUp size="14" />
```

Comment:
We tried to avoid hardcoding size values like that, you can probably find examples in the code, getting the size from the theme directly. Also it feels like the arrow isn't perfectly positionned, it's a bit too closed to the name and not perfectly vertically aligned (a bit too high)